tidy_data/tidy_data.md

```diff
@@ -0,0 +1,737 @@
+[[X]] "Height in meters" is an example of a variable.
+[[X]] Observations should each be in their own row
+[[?]] There are multiple correct answers!
+
+
+<div class = "answer">
+<details><summary>Click to see an explanation of the answer.</summary>
+
+An observation is a single thing with measurable characteristics, and each observation should be in its own row.  Variables, on the other hand, are things being measured, like height, color, score, count, and so on.  Variables should each be in their own column.  Values belong in cells, the intersection of rows and columns, and each cell should have one and only one value -- no combining!
+
+</details>
+</div>
+</div>
+
+## Sources of Messy Data
+
+Hadley Wickham, one of the authors mentioned in the last section, suggests in his [landmark work on tidy data](https://www.jstatsoft.org/article/view/v059i10) that there are five common problems that occur to make data "messy":
+
+* Column headers are values, not variable names.
+* Multiple variables are stored in one column.
+* Variables are stored in both rows and columns.
+* Multiple types of observational units are stored in the same table.
+* A single observational unit is stored in multiple tables.
+
+Let's take a look at each of these problems in turn.
+
+<div class = "hint">
+*Tidy Data* was published in 2014, and covers both the concepts around tidy data as well as the then state of the art code solutions in R for data reshaping.  In this module, we're not going to delve into the code examples provided by the article, for two reasons:
+
+* This module is intended to be code-free, making it useful for users of any programming language or none at all (this module can be useful just in terms of setting up data collection instruments well)
+* Since 2014, Hadley Wickham and others have developed new methods for data tidying
+
+So, if you choose to [read along in the article](https://www.jstatsoft.org/article/view/v059i10), concentrate on the principles, not the implementation details or code jargon.
+
+</div>
+
+### Column Headers as Values
+
+Let's consider an example from section 3.1 of [Hadley's article](https://www.jstatsoft.org/index.php/jss/article/view/v059i10/772).  The column headers in this table, taken from Pew Forum, contain values -- income values.  This table is readable for humans, and takes advantage of horizontal space on a page.  It makes sense that it would appear in a report for human consumption.  This data is messy, however, because it violates one of the important rules about tidy data.
+
+<div class = "hint"> Note that in the world of R users, Hadley Wickham is considered such a singularly important and yet friendly figure that he is often referred to simply as "Hadley".
+</div>
+
+| religion | <lia-keep><$10k</lia-keep> | <lia-keep>$10–20k</lia-keep> | <lia-keep>$20–30k</lia-keep> | <lia-keep>$30–40k</lia-keep> | <lia-keep>$40–50k</lia-keep> | <lia-keep>$50–75k</lia-keep> |
```

Comment:
liascript, in its enthusiasm, wants to automatically provide a plotting option for each table. To turn that off (so it displays only a table with no inviting option to plot the data), you have to add an attribute `data-type="None"`. https://liascript.github.io/course/?https://raw.githubusercontent.com/liaScript/docs/master/README.md#137
You'll likely want to do that for all the example data tables here; I'm seeing liascript plotting suggestions for all of them, many of which are very silly plots especially for the messier data.

Reply:
all fixed TYSMl